tox.ini

```
@@ -26,7 +26,7 @@ deps =
     {[base]deps}
     pytest
     ; Needed for `BBHx` package to work with PyCBC
-    ; git+https://github.com/mikekatz04/BBHx.git@4fff509; sys_platform == 'linux'
+    ; git+https://github.com/titodalcanton/BBHx/tree/py39-and-cleanup; sys_platform == 'linux'
```

I think this needs to be
```suggestion
    git+https://github.com/titodalcanton/BBHx.git@py39-and-cleanup; sys_platform == 'linux'
```